bpaingest/libs/common_resources.py

```diff
@@ -0,0 +1,4 @@
+import re
+# TODO: there's possibly some redundant escaping here: add tests for this before removing
+bsd_md5_re = re.compile(r'^MD5 ?\(([^\)]+)\) ?= ?([0-9a-f]{32})$')
```

Comment:
looks right to me, the escaping is to match literal parentheses...

Reply:
@grahame  The comment referred to the character inside the square brackets (for character). I don't think you need to escape asterisks, the bracket possibly too but I couldn't say for sure  so just flagged to leave for later